### PR TITLE
Extended usage docs in README.md

### PR DESCRIPTION
**Pull request by icamys**
Added docs on how to properly insert/upsert data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,35 @@ $client = new Client($conn, new PurePacker());
...
+// Inserting data
+$insertValues = [
+    'first insert val',
+    'second insert val',
+];
+$space->insert($insertValues);
+
+// Upserting values
+$upsertValues = [
+    1,
```
Comment (rybakit) [inline README.md]: Is it intended to have `1` here but not in the insert values? I'm curious how the space schema will look like for these examples.

Comment (icamys) [reply README.md]: Sorry, my bad. Updated PR

Comment (rybakit) [reply README.md]: If $insertValues has `1`, then $upsertValues must have `2`, otherwise you upsert examples are incorrect.

File: README.md [hunk 2]
```diff
@@ -35,9 +35,36 @@ $client = new Client($conn, new PurePacker());
...
+    'first upsert val',
+    'second upsert val',
+];
+
+// Values do not exist, inserting them
+// '=' - operator string, see full list of operators on official docs page
+// 0, 1 - inserted tuple indecies
```
Comment (rybakit) [inline README.md]: Where `0` comes from? I would delete this line.

**Comment by rybakit** [comment]
@icamys What do you think about simplifying php code samples to something like:

```php
$space->insert([1, 'foo', 'bar']);
...
$space->upsert([2, 'baz', 'qux'], [['=', 1, 'BAZ'], ['=', 2, 'QUX']]);
...
$space->upsert([2, 'baz', 'qux'], [['=', 1, 'BAZ'], ['=', 2, 'QUX']]);
```
?

**Comment by icamys** [comment]
@rybakit No objections

**Comment by rybakit** [comment]
@icamys Thanks!

**Pull request merged**